src/main/java/org/jabref/gui/push/PushToApplicationSettings.java

```diff
@@ -23,24 +23,24 @@ public PushToApplicationSettings(DialogService dialogService) {
         this.dialogService = dialogService;
     }
 
-    public GridPane getJFXSettingPane(int n) {
-        switch (n) {
-            case 0:
+    public GridPane getJFXSettingPane(String applicationName) {
```

Comment:
What about tests of the form `application is instanceof(...)`, then you don't need to work with "magic" strings.

Reply:
I found it very hard to understand the code in PushToApplicationSettings, so i took the liberty, to refactor this class a little bit.